math.js

```diff
@@ -28,3 +28,24 @@ function testAdd() {
 function subtract(a, b) {
   return a - b;
 }
+
+/**
+ * This will compute the result by raising number a to the power of b in decimal system.
+ * @param {number} a the base number
+ * @param {number} b the power value to raise.
+ * @returns The result.
+ */
+function power(a, b) {
+  // pre conditions
+  assert(typeof a === 'number');
+  assert(typeof b === 'number');
+  if (a === 0 || b === 0) {
+    return 0;
+  }
+
+  let product = a;
```

Comment:
- Initializing product with a results in bug.
- Instead initalizing product to 1 might resolve this bug

Reply:
resolved